src/Avalonia.Base/Controls/ResourceDictionary.cs

```diff
@@ -258,6 +258,24 @@ public bool TryGetValue(object key, out object? value)
             return false;
         }
 
+        /// <summary>
+        /// Ensures that the resource dictionary can hold up to <paramref name="capacity"/> entries without
+        /// any further expansion of its backing storage.
+        /// </summary>
+        /// <remarks>This method may have no effect when targeting .NET Standard 2.0.</remarks>
```

Comment:
This remark is partially outdated now.

Reply:
I've changed it to "may have" since it's still true that it doesn't do anything when on .NET standard and the inner dictionary is already initialized.